facia-press/app/controllers/Application.scala

```diff
@@ -23,6 +23,7 @@ class Application(liveFapiFrontPress: LiveFapiFrontPress, draftFapiFrontPress: D
 
   def generateLivePressedFor(path: String): Action[AnyContent] = Action.async { request =>
     liveFapiFrontPress.getPressedFrontForPath(path)
+      .map(_.full)
```

Comment:
Should we add a route like `/pressed/live/*path.lite` so we can get the lite version of the pressed front?

Reply:
Can do, I will add it